examples/nlp/token_classification/conf/ner_config.yaml

```diff
@@ -13,34 +13,25 @@ pl:
 model:
   data_dir: ??? # /path/to/data
   class_balancing: null # choose from [null, weighted_loss]. Weighted_loss enables the weighted class balancing of the loss, may be used for handling unbalanced classes
+  max_seq_length: 128
+  pad_label: 'O'
+  ignore_extra_tokens: false
+  ignore_start_end: false
+  overwrite_processed_files: false
+  batch_size: 32
+  num_workers:  2
```

Comment:
num_workers, pin_memory, and drop_last, batch size are for data loaders? Shouldn't they be there? Now the data loaders parts are very short. batch size can be different for train and evaluation as evaluation can be higher to speed up that part.

Reply:
fixed